rules/windows/execution_command_shell_started_by_svchost.toml

```diff
@@ -118,7 +118,7 @@ type = "new_terms"
 
 query = '''
 host.os.type:windows and event.category:process and event.type:start and process.parent.name:"svchost.exe" and 
-process.name.caseless:"cmd.exe"
+process.name:("cmd.exe" or "Cmd.exe") or ?process.pe.original_file_name:"Cmd.Exe"
```

Comment:
```suggestion
(process.name:("cmd.exe" or "Cmd.exe") or process.pe.original_file_name:"Cmd.Exe")
```

We need to keep the () to avoid non-intended matches
![image](https://github.com/elastic/detection-rules/assets/26856693/296000ee-5ffa-4b86-a7e9-2c241c45c961)